contrib/export/main.go

```diff
@@ -0,0 +1,103 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"strings"
+	"time"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	extypes "github.com/cosmos/cosmos-sdk/contrib/export/types"
+	"github.com/cosmos/cosmos-sdk/contrib/export/v036"
+	"github.com/spf13/cobra"
+	"github.com/tendermint/tendermint/types"
+)
+
+var migrationMap = extypes.MigrationMap{
+	"v0.36": v036.Migrate,
+}
+
+const (
+	flagSource      = "source"
+	flagGenesisTime = "time"
+	flagChainId     = "chain-id"
+)
+
+func migrateGenesisCmd() *cobra.Command {
```

Comment:
I've cleanup the implementation a bit. While doing so, I found the UX to be a very confusing. The v0.36 migration internally uses v0.34 types to migrate from...which is correct. But the confusion stems from having a `source` flag. Further thought leads me to believe this is unnecessary.

It really boils down to: `$ export genesis <target-version> <genesis-file> --genesis-time=... --chain-id=...`. I think this is all we need? The target version implicitly knows the source version (it's contiguous).

Reply:
I fully agree

Reply:
let's kill that imaginary need XD I agree was a predisposition for the future but we'll see it just next version if we want it or not